boards/common/e104-bt50xxa-tb/include/board.h

```diff
@@ -56,6 +56,14 @@ extern "C" {
 #define BTN1_MODE           GPIO_IN_PU
 /** @} */
 
+/**
+ * @name    WS281x RGB LED configuration
+ * @{
+ */
+#define WS281X_TIMER_DEV    TIMER_DEV(1)            /**< Timer device */
+#define WS281X_TIMER_MAX_VALUE TIMER_1_MAX_VALUE    /**< Timer max value */
```

Comment:
Why not always use `TIMER_DEV(1)` for `WS281X_TIMER_DEV`?

Reply:
```c
/*
...
 * The default value of 2 is chosen because the only platform on which the
 * module is usable is nRF5x, where TIMER_DEV(1) is in use by the radio module.
 * It is strongly advised to explicitly set this timer to a known free timer,
 * as the default may change without notice.
*/
```

Hmmm reading that I wonder if there will be a conflict with the e104-bt50xxa-tb boards.  But I guess that is only if both modules are used at the same time.

Reply:
The test doesn't use the radio module, so it can select `TIMER_DEV(1)` - and we don't have to clutter the board definitions with `WS281X_TIMER_DEV` when there is no `ws281x` LED connected.

Reply:
> The test doesn't use the radio module, so it can select TIMER_DEV(1)

I still think it is better try not to conflict with a known value.  If we add a radio and ws281x test it would fail.  This is maybe a decision best left to @chrysn though...